extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -38,7 +40,22 @@ const renderShoppingTagTemplate = (tagData) => (
         }
       ></span>
       <span class="amp-story-shopping-tag-pill-text">
-        {tagData['product-tag-text'] || '$' + tagData['product-price']}
+        {(tagData['product-tag-text'] && (
+          <span class="amp-story-shopping-product-tag-text">
+            {tagData['product-tag-text']}
+          </span>
+        )) ||
+          new Intl.NumberFormat(
+            localize(
+              element,
+              LocalizedStringId_Enum.AMP_STORY_SHOPPING_LANGUAGE_ISO_LABEL
+            ),
+            {
+              style: 'currency',
+              currency: tagData['product-price-currency'],
+              maximumFractionDigits: 0 /* This is needed to NOT display the Trailing zeroes (for now), otherwise by default $100 will display as $100.00*/,
```

Comment:
Google shopping shows trailing zeros even if it's `.00`. 
If we keep this, decimal values wont show.
I think we should remove `maximumFractionDigits ` so that a) decimal values will work and b) UX aligns with google shopping.

We can revisit this later if needed, but I think it's better to remove this to ensure decimal places will work.
Please also add decimal places in the example. It will better match real-world application.